### ISSUE TITLE
'bert-base' for ProteinBertForMaskedLM does not seem to work

### ISSUE THREAD

**Issue by RensDimmendaal** [songlab-cal/tape#14]
I'm trying to predict masked tokens. However, the predictions are of low quality.
As a sanity check I'm not masking any tokens and look if the model predicts back the original tokens with reasonable accuracy (see code below).
However, I'm encountering two issues:

1. The predictions vary, even when the model is put in eval() mode
2. The predictions are of very low quality (near 0% accuracy)

Can you please tell me if loading in the 'base-bert' for language modeling not supported yet or otherwise if am I doing something wrong?

Thanks!

```python
import numpy as np
import torch
from tape import TAPETokenizer,  ProteinBertForMaskedLM

tokenizer = TAPETokenizer(vocab='iupac')  # iupac is the vocab for TAPE models, use unirep for the UniRep model

model = ProteinBertForMaskedLM.from_pretrained('bert-base')
model.eval() # deactivate dropout


# Pfam Family: Hexapep, Clan: CL0536
sequence = 'GCTVEDRCLIGMGAILLNGCVIGSGSLVAAGALITQ'
token_ids = tokenizer.encode(sequence)
input_tensor = torch.tensor([token_ids])

# Predict all tokens
with torch.no_grad():
    outputs = model(input_tensor)
    predictions = outputs[0]
    logits = predictions[0] # only one sequence
    
pred_ids = logits.detach().numpy().argmax(1)
pred_tokens = tokenizer.convert_ids_to_tokens(pred_ids)
pred_seq = tokenizer.convert_tokens_to_string(pred_tokens)
print(pred_seq)
```

**Comment by rmrao** [comment]
Ah sorry we currently have just the pretrained model w/o the actual finetuning head up. I can probably upload that as well, if you think that would be useful.

**Comment by RensDimmendaal** [comment]
I would definitely use it. I expect it will help with tuning for domain specific datasets, and for some generative modeling.

Also, thanks for moving the library over to hugging face! I already moved over your weights to their tensorflow implementation so it gets easier to add the vectors into a keras model now.

**Comment by rmrao** [comment]
Just uploaded the full weights so if you re-download you should get the correct result. I haven't implemented force-redownload yet (see issue #6), so you may have to manually delete the cache file.

If you're on linux this is in `~/.cache/torch/protein_models` and if you're on windows there's a short snippet of code that should find it (I posted it in #1). Will add the force download option soon so this can be avoided.

**Comment by rmrao** [comment]
Oh actually it should detect a change in the upstream file and download it automatically, so you shouldn't have to worry about it.

**Comment by RensDimmendaal** [comment]
Thanks a lot it works perfectly.
